WordPress/Classes/ViewRelated/Pages/PageListViewController.swift

```diff
@@ -546,14 +546,9 @@ class PageListViewController : AbstractPostListViewController, UIViewControllerR
         
         WPAnalytics.track(.PostListOpenedCellMenu, withProperties: propertiesForAnalytics())
         
-        if !UIDevice.isPad() {
-            presentViewController(alertController, animated: true, completion: nil)
-            return
-        }
-        
+
         alertController.modalPresentationStyle = .Popover
         presentViewController(alertController, animated: true, completion: nil)
-        
```

Comment:
No need to stick all the code together in a big block.  This spacing was nice for readability.